docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,575 @@
+:::
+::::
+
+### Creating a new localized entry
+
+To create a localized entry from scratch, send a POST request to the Content API.
+
+If no locale has been passed in the request body, the entry is created using the default locale for the application:
+
+:::: tabs
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants
+```
+
+```json
+{
+  "name": "Restaurant B",
+  "slug": "restaurant-b",
+  "stars": 1,
+  "description": "Restaurant with terrace in front of the lake"
+}
+```
+
+:::
+
+::: tab Response
+
+**Example response**
+
+```json
+{
+    "id": 5,
+    "name": "Restaurant B",
+    "slug": "restaurant-b",
+    "price_range": null,
+    "contact_email": null,
+    "stars": 1,
+    "averagePrice": null,
+    "address": null,
+    "timestamp": null,
+    "short_description": null,
+    "since": null,
+    "description": "Restaurant with terraza in front of the lake",
+    "menu": null,
+    "locale": "en",
+    "published_at": "2021-04-07T12:51:43.681Z",
+    "created_at": "2021-04-07T12:51:43.695Z",
+    "updated_at": "2021-04-07T12:51:43.695Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": []
+}
+```
+
+:::
+::::
+
+To create a localized entry for a locale different from the default one, add the `locale` attribute to the body of the POST request:
+
+:::: tabs
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants
+```
+
+```json
+{
+  "name": "Restaurant F",
+  "slug": "restaurant-f",
+  "stars": 1,
+  "description": "Restaurant face à la mer",
+  "locale": "fr"
+}
+```
+
+:::
+
+::: tab Response
+**Example response**
+
+```json
+{
+    "id": 8,
+    "name": "Restaurant F",
+    "slug": "restaurant-f",
+    "price_range": null,
+    "contact_email": null,
+    "stars": 1,
+    "averagePrice": null,
+    "address": null,
+    "timestamp": null,
+    "short_description": null,
+    "since": null,
+    "description": "Restaurant face à la mer",
+    "menu": null,
+    "locale": "fr",
+    "published_at": "2021-04-07T13:22:46.589Z",
+    "created_at": "2021-04-07T13:22:46.617Z",
+    "updated_at": "2021-04-07T13:22:46.617Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": []
+}
+```
+
+:::
+::::
+
+### Creating a localization for an existing entry
+
+To create another localization for an existing localized entry, send a POST request to the appropriate URL depending on the type of content:
+
+| Content-Type    | Request URL format                       |
+|:----------------|:-----------------------------------------|
+| Single Type     | `POST /{content-type}/:id/localizations` |  
```

Comment:
I feel like there is something weird there.
For a Single Type, we presume that there is one entry.
I don't really understand why we need to precise the `:id` for Single Type.
Isn't it for Collection Type instead? Lines are propably reverted.

Reply:
Indeed, something wrong over there…
Need to check with @Convly

Reply:
They currently had some bugs with this and are fixing them I think.